src/main/webapp/app/guided-tour/guided-tour-settings.model.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Guided tour settings model which can dynamically handle the settings for
+ * showCourseOverviewTour | showNavigationTour | showProgrammingExerciseTour | showQuizExerciseTour | showModelingExerciseTour | showTextExerciseTour
+ */
+export class GuidedTourSettings {
+    [key: string]: boolean;
```

Comment:
I have never seen this? Is this an actual map?

Reply:
Yes, this is a TypeScript Map.